client/src/pages/organizations/Show.js

```diff
@@ -373,6 +401,18 @@ const OrganizationShow = ({ pageDispatchers }) => {
                   humanValue={Organization.humanNameOfType}
                 />
 
+                {showLeadingPositions(
+                  organization.positions,
+                  PositionRole.LEADER.toString(),
+                  "Leaders"
+                )}
+
+                {showLeadingPositions(
+                  organization.positions,
+                  PositionRole.DEPUTY.toString(),
+                  "Deputies"
```

Comment:
Don't hard-code this, use something like `pluralize(utils.titleCase(PositionRole.DEPUTY.humanNameOfRole()))` instead.

Reply:
OK